osd_cluster_admin/osd_hibernate_resume/osd-overview-hibernating-resuming-cluster.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+// * osd_cluster_admin/osd_hibernate_resume/osd-overview-hibernating-resuming-cluster.adoc
+
+:_content-type: CONCEPT
+[id="osd-overview-hibernating-resuming-cluster_{context}"]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+include::_attributes/common-attributes.adoc[]
+= Overview of hibernating and resuming cluster (Technology Preview)
+
+:context: osd-overview-hibernating-resuming-cluster
+
+toc::[]
+
+As a cluster administrator, you might need to hibernate a {product-title} cluster for a period and resume it later to conserve resources.
+
+[IMPORTANT]
+====
+Hibernating a ready cluster and resuming a hibernating cluster are Technology Preview features only. The hibernate and resume features for {product-title} clusters are provided as Technology Preview features as explained in link:https://access.redhat.com/articles/7012966[Technical Preview: Hibernating and Resuming Clusters]. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. 
+
+During the entire period of hibernation and till the resumed cluster is ready, the cluster is in link:https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-limited-support_rosa-service-definition[Limited Support] status under {product-title} SLA.  For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]
+====
+
+A hibernating cluster requires significantly fewer resources than one that is running, potentially leading to lowered provider costs through the strategic movement of clusters in and out of a hibernating state. Hibernating a ready cluster on-demand can notably reduce infrastructure expenses, including those related to EC2 instances and network devices. 
```

Comment:
I'd suggest to write the facts instead of using languages like significantly fewer resources etc. For example, 'Hibernation shuts down the underlying Amazon EC2 instances that are part of the cluster, allowing for saving infrastructure costs in case of using on-demand EC2 instances or conserving RI purchases.'

Reply:
Updates as, "Hibernation shuts down the underlying Amazon Elastic Compute Cloud (EC2) instances that are part of the cluster. This allows for saving infrastructure costs in the case of using on-demand EC2 instances or conserving reserved instances (RI) purchases."